### ISSUE TITLE
Share popup for draw v2 is incorrect on Firefox

### ISSUE THREAD

**Issue by Jenselme** [geoadmin/mf-geoadmin3#3513]
The input fields to share the link are small and I don't see the _copy link_ buttons (see capture below). I tested on FireFox Nightly and 49 on Linux.
![screen shot 2016-09-29 at 09 59 58 resized](https://cloud.githubusercontent.com/assets/6643536/18945755/3d416888-862c-11e6-8a92-31db0df0f8e9.png)

On chrome:
![screencapture-map-geo-admin-ch-1475136119039 resized](https://cloud.githubusercontent.com/assets/6643536/18945764/44b4491e-862c-11e6-8366-ba70b7c8e824.png)

**Comment by Jenselme** [comment]
I think you should. As far as I know it should work and it is good from a user perspective.

**Comment by loicgasser** [comment]
Seems like document.execCommand('copy') is available on all browsers, including IE9.

https://developer.mozilla.org/en/docs/Web/API/Document/execCommand

Feature	 Chrome	Firefox (Gecko)	Internet Explorer	Opera	Safari
copy/cut	  42	        41 (41)	                  9	                         29	         10

**Comment by davidoesch** [comment]
+1

**Comment by oterral** [comment]
it doesn't work on firefox 43 execCommand('copy') returns false.

**Comment by loicgasser** [comment]
document.execCommand(‘cut’/‘copy’) was denied because it was not called from inside a short running user-generated event handler.

That's what I get.

Found the bug mentionning that:
https://bugzilla.mozilla.org/show_bug.cgi?id=1197451


And from https://www.w3.org/TR/clipboard-apis/ I got
If an implementation supports ways to execute clipboard commands through scripting, for example by calling the document.execCommand() method with the commands "cut", "copy" and "paste", the implementation must trigger the corresponding action, which again will dispatch the associated clipboard event.

These are the steps to follow when triggering copy, cut or paste actions through a scripting API:

Execute the corresponding action synchronously.
Use the action's return value as the return value for the API call.
